scripts/build-and-test-package-full-network-outage-retry-tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -x
+set -e
+
+ROOT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )"/.. &> /dev/null && pwd )
+
+echo "Building and testing package: ${1}"
+
+# Start the Docker container in the background
+DOCKER_CONTAINER_NAME="momento-local-test"
+docker run --name ${DOCKER_CONTAINER_NAME} -d -p 8080:8080 gomomento/momento-local \
+    --return-error unavailable --error-rpcs get
+
+# Ensure the container is stopped when the script exits
+function cleanup {
+    echo "Stopping and removing Docker container..."
+    docker stop ${DOCKER_CONTAINER_NAME}
+    docker rm ${DOCKER_CONTAINER_NAME}
+}
+trap cleanup EXIT
```

Comment:
In Github Actions, the native way of running this is to set the services property in the job. See [here](https://github.com/momentohq/momento-java-lettuce-client/blob/main/.github/workflows/on-pull-request.yml#L27-L31). We can set the various docker settings there as well.

We will need a script like this as a prerequisite to run locally.